src/primitive_debug.cc

```diff
@@ -22,34 +22,28 @@ namespace toit {
 
 MODULE_IMPLEMENTATION(debug, MODULE_DEBUG)
 
+struct PerClass {
```

Comment:
I don't like the PerClass name much. 

I think it should contain the name 'instances' somehow because that is what we're accumulating information about.

InstancesData?